From a447c699150ed371e0a2e13257d040a4d6576e65 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Wed, 12 Oct 2022 19:49:49 +0200
Subject: [PATCH 230/841] feat(stm32mp2-fdts): add UART console on
 stm32mp257f-ev boards

Add pins and node for USART2 to support console on stm32mp257f-ev boards.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 5803a15fc6303b0d5e0dafc095907eacabcf34fd feat(stm32mp2-fdts): add UART console on stm32mp257f-ev boards]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Icd5c1c867edb1c19cd6d85d9f9c34b39d01a122d
---
 fdts/stm32mp257f-ev1.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index 58eb1cf23..23f61e5fb 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -42,3 +42,9 @@
 	bus-width = <4>;
 	status = "okay";
 };
+
+&usart2 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&usart2_pins_a>;
+	status = "okay";
+};
-- 
2.47.2

